index.html

```diff
@@ -0,0 +1,118 @@
+        </li>
+      </ul>
+    </div>
+    <!-- start sectionone -->
+    <section class="sectionone pd-y" id="sectionone">
+      <div class="sectionone__wrapper">
+        <h1 class="" id="">Hey I am Mimic</h1>
+      <small><h3 class="center">a web developer</h3></small>
+      </div>
+    </section>
+    <!-- end sectionone -->
+    <!-- <h1>hello world and I am Mimic</h1>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis
+      laudantium excepturi dolorem rerum labore repellat voluptas debitis animi
+      culpa iusto!
+    </p> -->
+    <section>
+      <div class="flexheader pd-y" id="flexheader">
+        <p class="center">these are some of my projects</p>
+
+        <div class="flex">
+          <a href="#">
+            <img
+              src="https://icatcare.org/app/uploads/2018/07/Thinking-of-getting-a-cat.png"
+          /></a>
+          <a href="#">
+            <img
+              src="https://www.rd.com/wp-content/uploads/2021/01/GettyImages-1175550351.jpg"
+          /></a>
+          <a href="#">
+            <img
+              src="https://cdn.britannica.com/39/7139-050-A88818BB/Himalayan-chocolate-point.jpg"
+          /></a>
+          <a href="#">
+            <img
+              src="https://www.boredpanda.com/blog/wp-content/uploads/2022/06/funny-pics-cats-with-powerful-auras-cover_800.png"
+              alt="this is image four"
+          /></a>
+          <a href="#">
+            <img
+              src="https://mediaproxy.salon.com/width/1200/height/675/https://media.salon.com/2022/05/cats-party-0516221.jpg"
+          /></a>
+          <a href="#">
+            <img
+              src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRzklJZyjjFXAqKOZ3qhEObLEXPTxRrEc16Dw&usqp=CAU"
+          /></a>
+          <a href="#">
+            <img
+              src="https://moderncat.com/wp-content/uploads/2014/03/tortoiseshell-kitten-closeup-p-46771525-940x640.jpg"
+          /></a>
+        </div>
+      </div>
+    </section>
+    <!-- start sectiontwo -->
+    <section class="sectiontwo pd-y" id="sectiontwo">
+      <div class="sectiontwo__wrapper">
+        <h1 class="center">Let's work together...</h1>
+        <p class="center">How do you take your coffee?</p>
+        <div>
+          <li><a href="#">Facebook</a></li>
+          <li><a href="#">GitHub</a></li>
+          <li><a href="#">Twitter</a></li>
+          <li><a href="#">Send a mail</a></li>
+          <li><a href="#">call me</a></li>
+        </div>
+      </div>
+    </section>
+    <!-- end sectiontwo -->
+    <!-- start footer -->
+    <footer class="footer" id="footer">
+      <span class="float" id="footing"
+        >**This is just a fake portfolio. All the projects and contact details
+        given are not real.
+      </span>
+      <span class="float-r" id="footing">© Created for freeCodeCamp </span>
+    </footer>
+    <!-- end footer -->
+    <script>
+      console.log("hello world")
```

Comment:
Delete the console from the codebase, it is not necessary to submit the code with the console.

Reply:
You have rightly said that code with a console log should not be submitted. But be polite in conveying this message.
Furthermore, it would also be better if the coined language is used, so that all the necessary details regarding **Not Using Console Log** are clear. For Example: `Do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.`